C7GameData/GameData.cs

```diff
@@ -105,23 +100,25 @@ public Player CreateDummyGameData()
 
             return humanPlayer;
         }
-
-        private void CreateStartingDummyUnits(Player humanPlayer)
+        private void CreateDefaultUnitPrototypes()
```

Comment:
protip: you can do this with a lot less code using collection initialization:

`unitPrototypes = new Dictionary<string, UnitPrototype>()
{
  { "warrior", new UnitPrototype { name="warrior", attack=1, defense=1, movement=1, iconIndex=6, shieldCost=10 }
  ...
}`

Reply:
Nifty!  That's way better than the style Rider was suggesting.  I've switched it over, and fixed Chariots being too expensive at the same time.